src/feature/AlignHashRocketsFeature.ts

```diff
@@ -0,0 +1,125 @@
+import { IFeature } from '../feature';
+import * as vscode from "vscode";
+import { ILogger } from "../logging";
+import { IAggregateConfiguration } from "../configuration";
+
+class HashRocketLocation {
+  public rocketIndex: number;
+  public preceedingTextIndex: number;
+}
+
+class AlignHashRocketsFormatter implements vscode.OnTypeFormattingEditProvider{
+  private logger: ILogger;
+
+  constructor(logger: ILogger) {
+    this.logger = logger;
+  }
+
+  private extractHashRocketLocation(document: vscode.TextDocument, line: number): HashRocketLocation {
+    if (line < 0 || line > document.lineCount) { return null; }
+
+    var textLine = document.lineAt(line);
+    // Find the first hash-rocket `=>`
+    var hashRocket = textLine.text.indexOf(' =>');
```

Comment:
There might not always be a space before the sign

Reply:
there should be :-)